server/controllers/stripe.js

```diff
@@ -0,0 +1,37 @@
+const User = require("../models/User");
+const Profile = require("../models/Profile");
+const asyncHandler = require("express-async-handler");
+const dotenv = require('dotenv').config({path:__dirname+'/./../.env'})
+const stripe = require('stripe')(process.env.STRIPE_SECRET_KEY);
+
+exports.createUser = asyncHandler(async (req, res, next) => { 
+  const user = await User.findById(req.user.id);
+  const profile = await Profile.findOne(req.user);
+
+  const account = await stripe.accounts.create({
+    email: user.email,
+    business_type: "company",
+    company: {
+      name: profile.name,
+    },
+    country: 'CA',
+    default_currency: 'cad',
+    type: 'standard', 
+    });
+
+    profile.stripeAccountId = account.id;
+    await profile.save()
```

Comment:
Hm - just thinking about this now. They could have their Stripe account created but their link expires. Should we check for this as well? I might make this a new ticket as I think about how to go about doing this.

Reply:
Sounds good - will give this some more thought too!